testsuite/master/verify-data/plugin.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @name data-test
+ * @version 1.0.0
+ * @api 4.0.0
+ * @main DataTest
+ * @author PMMP Team
+ */
+
+class DataTest extends \pocketmine\plugin\PluginBase {
+	public function onEnable() : void {
+		$data = file_get_contents($this->getDataFolder() . "/create-data");
+		if($data !== "successful") {
+			$this->getLogger()->critical("create-data is not successful");
+			exit(1);
```

Comment:
The `exit(1);` here doesn't really exit the process, but just causes the crashdump watchdog to hang the process. Is there a better method to reliably error out?